.editorconfig

```diff
@@ -4,11 +4,12 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-indent_size = 4
+indent_size = 2
 indent_style = space
 insert_final_newline = true
 max_line_length = 80
 trim_trailing_whitespace = true
+quote_type = "single"
```

Comment:
This conflicts with the ca-style-guide gem. Not sure if that matters too much though.